subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -755,12 +755,6 @@ The following code enables JUnit Platform support in `build.gradle`:
 
 See api:org.gradle.api.tasks.testing.Test#useJUnitPlatform()[] for more details.
 
-[NOTE]
-====
-There're some known limitations on JUnit 5, e.g. tests in static nested classes won't be discovered and classes are still displayed by its name instead of `@DisplayName`.
-They'll be fixed in future version of Gradle. If you find more, please don't hesitate to tell us: https://github.com/gradle/gradle/issues/new
-====
```

Comment:
I'm assuming that this was fixed in some other work but this was never removed from the documentation?